tmva/pymva/inc/TMVA/MethodPyAdaBoost.h

```diff
@@ -19,24 +19,23 @@
 //                                                                      //
 // MethodPyAdaBoost                                                     //
 //                                                                      //
-//                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
+#ifndef ROOT_TMVA_PyMethodBase
```

Comment:
Not sure about all the details of the coding style, but at least here I think it should be normal header guards in the respective header and no "double guards"around the include :)